docs/_docs/windows.md

```diff
@@ -212,5 +212,7 @@ As of v1.3.0, Jekyll uses the `listen` gem to watch for changes when the `--watc
 Add the following to the Gemfile for your site if you have issues with auto-regeneration on Windows alone:
 
 ```ruby
-gem 'wdm', '~> 0.1.0' if Gem.win_platform?
+gem 'wdm', '~> 0.1.1' if Gem.win_platform?
```

Comment:
It makes sense though if at least version `0.1.1` is needed. Given that in this version the installation with Ruby 2.2 apparently has been fixed ([see wdm changelog](https://github.com/Maher4Ever/wdm/releases/tag/v0.1.1)) i think this update to the docs makes sense.